tests/strands/models/test_bedrock.py

```diff
@@ -119,6 +119,7 @@ def test__init__with_region_and_session_raises_value_error():
 
 def test__init__default_user_agent(bedrock_client):
     """Set user agent when no boto_client_config is provided."""
+    _ = bedrock_client
```

Comment:
Can we turn this into an autofixture instead or use `usefixtures`?  E.g. is there a reason we need to have it as a parameter or use